include/zephyr/dt-bindings/clock/mcux_lpc_syscon_clock.h

```diff
@@ -58,6 +58,7 @@
 #define MCUX_FLEXSPI2_CLK		37
 
 #define MCUX_MRT_CLK			40
+#define MCUX_FREEMRT_CLK		401
```

Comment:
Why use this ID? I guess the intention here is to try and group IP that is similar? I think this is a good idea, but we should be consistent. Would you be willing to add a zero on all the IDs declared in this file? I think that would allow us to effectively expand definitions like this in the future

Reply:
dont think it matters what the numbers are as long as they are different which now looking at this file I just realized for some reason someone made the ctimer values the same as the flexcomms

Reply:
Yeah it shouldn't matter. I see what you mean about the collision between the ctimer and flexcomm. This is what I'm pointing out, I think that making these IDs have a "high byte" and "low byte" like we do in the CCM driver helps prevent these collisions: https://github.com/zephyrproject-rtos/zephyr/blob/main/include/zephyr/dt-bindings/clock/imx_ccm.h#L66

Reply:
not doing it here

Reply:
Fair enough- I made a PR to implement it here: https://github.com/zephyrproject-rtos/zephyr/pull/70254